plugins/localization/package.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "@wecre8websites/builder-localization",
```

Comment:
`@wecre8websites/builder-localization `, this should start with `@builder.io` so we can release it to npm, also I think this extension is more for updating preview URL per locale, what do you think about naming it something closer to that, something like `Localized preview plugin`

Reply:
Updated: @builder.io/plugin-localized-preview